v2/testdata/eeServerCertValidEqual397.pem

```diff
@@ -0,0 +1,10 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Should these test files include the openssl dump (aka -text)

The other test certs seem to.

Reply:
I think I'm guilty of forgetting to do this with some frequency :sweat:

Fixed for this branch in 961cc9c. I cleaned up the utility script and will push a separate PR that adds the same text output to all the test files that are missing it in master.